Tasks/MSBuild/Helpers.ps1

```diff
@@ -60,6 +60,11 @@ function Invoke-BuildTools {
         [switch]$NoTimelineLogger
     )
 
+    if ($NuGetRestore)
```

Comment:
Our intention was to print the warning and continue with the existing behavior for now, yes.